server.py

```diff
@@ -1,12 +1,20 @@
 import os
-from flask import Flask, jsonify
+from flask import Flask, jsonify, request, url_for, make_response
 from flask_api import status
+from models.shopcart import Shopcart
 
 app = Flask(__name__)
-
 DEBUG = (os.getenv('DEBUG', 'False') == 'True')
 PORT = os.getenv('PORT', '5000')
 
+# Status Codes
```

Comment:
No need to have these HTTP codes. Please use `status` from `flask_api`

Reply:
I think we should keep the codes since there's no harm of having them and they'll help us remember what code corresponds to what response in the future. I personally dont know them by memory and will appreciate them in our API.